pkg/ui/ccl/src/views/clusterviz/containers/map/nodeView.tsx

```diff
@@ -22,12 +22,13 @@ import { CapacityArc } from "src/views/clusterviz/components/nodeOrLocality/capa
 import { Sparklines } from "src/views/clusterviz/components/nodeOrLocality/sparklines";
 import { LongToMoment } from "src/util/convert";
 import { cockroach } from "src/js/protos";
+
 import NodeLivenessStatus = cockroach.storage.NodeLivenessStatus;
-import Liveness$Properties = cockroach.storage.Liveness$Properties;
+type Liveness$Properties = cockroach.storage.Liveness$Properties;
 
 interface NodeViewProps {
   node: NodeStatus$Properties;
-  livenessStatus: { [id: string]: NodeLivenessStatus };
+  livenessStatuses: { [id: string]: NodeLivenessStatus };
```

Comment:
this one still takes one collection and one scalar?  weird, but maybe we just leave it for Later (TM).